proto/tendermint/privval/types.proto

```diff
@@ -62,6 +62,7 @@ message SignVoteRequest {
   string                chain_id    = 2;
   int32                 quorum_type = 3;
   bytes                 quorum_hash = 4;
+  tendermint.types.StateID state_id  = 5;
```

Comment:
Why would you need this? If the vote is being signed then the state would be known.